less/variables.less

```
@@ -1,14 +1,17 @@
-
-// Color assignments 
+// Color assignments
 @font-color: @shark;
 @button-bg: @eastern-blue;
-@main-bg:  @white;
+@main-bg: @white;
 @navigation-border: @silver;
 @footer-bg: @sandy-beach;
 
 // Colors
-@white: #FFFFFF;
-@silver: #C0C0C0;
+@white: #ffffff;
+@silver: #c0c0c0;
 @shark: #212529;
-@eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@eastern-blue: #17a2b8;
+@sandy-beach: #ffebcd;
+
+// Escape variable for mobile
+
+@mobile: (max-width: 500px);
```

Task 5: Mobile Updates Needed
 -  Match the design file at 500px as well as you can - DONE!
```suggestion - Use escaping to create a variable named @mobile that contains this value: (max-width: 500px).

> With escaping, you can use variables to hold a string value. The syntax looks like this ~"some string value". Let’s see it in action using our previous example with escaped values for the media queries.
> 
> // Escaped values
> @tablet: ~"(min-width: 500px)";
> @desktop: ~"(min-width: 800px)";
> 
> h1 {
>   font-size: 2rem;
>   padding: 10px;
> 
>   @media @tablet {
>     padding: 20px;
>   }
> 
>   @media @desktop {
>     font-size: 1.8rem;
>     padding: 0 10px;
>   }
> }

```